etc/parameters/default.json

```diff
@@ -1,15 +1,15 @@
 {
-  "object_detection": {
-    "object_detection_top": {
-      "enable": false,
-      "intersection_over_union_threshold": 0.45,
-      "distance_to_referee_position_threshold": 2.0,
-      "keypoint_confidence_threshold": 0.2,
-      "shoulder_angle_threshold": 0.2,
-      "foot_z_offset": 0.05,
-      "referee_pose_queue_length": 8,
-      "minimum_number_poses_before_message": 5
-    }
+  "pose_detection": {
+    "enable": false,
+    "intersection_over_union_threshold": 0.45,
+    "distance_to_referee_position_threshold": 2.0,
+    "bounding_box_confidence_threshold": 0.8,
+    "overall_keypoint_confidence_threshold": 0.7,
+    "visual_referee_keypoint_confidence_threshold": 0.9,
+    "shoulder_angle_threshold": 0.2,
```

Comment:
Just a general thought, I think it would be easier to understand what these parameters do if their names indicated whether they are minimum or maximum thresholds...

Reply:
Done